handlebarsHelpers.js

```diff
@@ -43,5 +43,15 @@ module.exports = {
     });
 
     return Object.values(speakers);
+  },
+  'getSponsors': function(events) {
+    const sponsors = {}
+    events.forEach(event => {
+      event.sponsors.forEach(sponsor => {
+        sponsors[sponsor.name] = sponsor;
+      });
+    });
+
+    return Object.values(sponsors);
```

Comment:
`Style` There's nothing wrong with this function but it duplicates the `getSpeakers` function differing only in that it is sponsors and not speakers.

I'm okay with the duplication 'cos the algorithm expressed in the functions is straightforward... but we are a tech meetup that's about JS. As such, maybe we should not have such dupes: extracting a shared function that can be called by both is easy enough.

Thoughts?

Reply:
💯  Agree! I'll refactor! :)